pkg/data/components/NopExporter.yaml

```diff
@@ -0,0 +1,22 @@
+kind: NopExporter
+name: DefaultNopExporter
+version: v0.1.0
+summary: Nop exporter
+description: |
+  Simple nop exporter.
+  This exporter does nothing, is required for minimal collector.
+ports:
+  - name: Traces
```

Comment:
i could see this being useful for additional signals, when someone wants to play around with configuration of their pipelines but dont want to emit anything just yet

Reply:
Agreed, I think we can do more with it. Right now it's basically the smallest possible component 😄